Gemfile

```diff
@@ -15,7 +15,7 @@ gem "middleman", "~> 4.3.7"
 gem "middleman-search_engine_sitemap", "~> 1.4"
 
 gem "github-markdown"
-gem "html-pipeline", "~> 2.5.0"
+gem "html-pipeline"
```

Comment:
Minor: could we link to the fix in the commit?

Reply:
I did try to track down the exact version/fix but couldn't find it.

Reply:
I might be missing something here: how do we know it's fixed?

Reply:
Ha, your comment made me go back and run the build - everything passes. So then I switched it back to 2.11.0 (the [buggy version referenced in this issue](https://github.com/alphagov/govuk-developer-docs/issues/1821)) to confirm it was broken in that version, and it was.

So then I proceeded to increment the version and run the build, to see which version fixed the issue.
I worked my way through all of the versions and found that _none_ of them work.
Then I realised the `Gemfile.lock` was still pointing to the pinned 2.5.0 version when the build last passed 🤦 

So it does appear to still be an issue. Tomorrow I'll have a go at delving in and fixing this specific issue.

Reply:
Ah, I feel validated for my awkward question 😸.

Reply:
I've now found the subdependency that was causing the error, and thought we might have to [expand the context options passed to HTML::Pipeline](https://github.com/jch/html-pipeline/pull/333), but on second thoughts it's easier and more consistent to just do the UTF-8 conversion here.